### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Disk_DiskInfoTable_ScreenReader: NVDA is not announcing any status after performing the sorting operation on all columns of the table.

### ISSUE THREAD

**Issue by shreyashi227** [microsoft/AzureStorageExplorer#3470]
**GitHubTags:** #C&AI_Aug2020_BMAzure Tools Storage Explorer;#A11yWCAG2.1;#A11YTCS;#DesktopApp;#Azure Tools Storage Explorer;#WCAG4.1.3;#Win32;#A11ySev2;#Benchmark;#NVDA;#JAWS;#FTP;#STP;

### **Environment Details:**
Application Name: Azure Tool Storage Explorer
Application version: 1.14.2
Windows Version - Windows10
Screen Reader : JAWS(2020.2006.12ILM),NVDA(2020.2)

### **Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Disks" of the selected subscription.
5. Navigate to "A11ytest" and  click on it.
6. Navigate to Table under the tool bar.
7. Start NVDA and navigate to the column headers of the table
8. Check if NVDA is not announcing any status after performing the sorting operation on all columns of the table.

### **Actual:**
NVDA is not announcing any status like "column is sorted in ascending or descending order" after performing the sorting operation on all columns of the table.

**This issue is also observed with JAWS**

This issue is observed in the below scenarios:
1. Blob Container Table - Table
2. FileShares - Filesharetable
3. Disk - DiskInfoTable
4. Queues

### **Expected:**
NVDA should announce any status like "column is sorted in ascending or descending order" after performing the sorting operation on all columns of the table.

### **User Impact:**
Screen reader users will not be able to understand whether is column is sorted in ascending or descending order successfully.

### **Attachment:**
https://microsoftapc-my.sharepoint.com/:v:/g/personal/v-shrkh_microsoft_com/EWWIlUhZ1dpJqOwpayqLBz8BjMTw2l_v0dD-QJEJlttFDg?e=dop7ha

**Comment by craxal** [comment]
@shreyashi227 Can you share a reference to the exact recommendation this issue is in violation of?

WCAG 4.1.3 defines success recommendations for _status messages_. The sorting state of a column does not appear to fall within this definition, because it does not alter any text or "success" status indication, and therefore it does not need to be announced. In addition, I've noticed that screen readers do not announce any changes of sort state in the example tables at https://www.w3.org/TR/wai-aria-practices/examples/grid/dataGrids.html.

**Comment by craxal** [comment]
@12-shweta You said:

> 3470 -> This bug is totally different. It is related to WCAG4.1.3 that is Status message.

As I mentioned above, the sort state of a table column is _not_ a _statue message_. (I'm reading from  https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=413#status-messages). The sort state of a column does not fall into any of the three situations listed. Furthermore, it says using alerts for information that is _not important or time sensitive_ is a _failure_. Unless I'm misunderstanding, or there is another recommendation that is being violated here, I don't think this is actionable.

**Comment by shreyashi227** [comment]
As per the discussion with SME, closing this bug as a known issue.

**Comment by fsteffi** [comment]
**Github Tags:**#A11yMAS;
